src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  margin: 0;
+  list-style-type: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  --color-active-blue: #00acdc;
+}
+
+.logo-container {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding instead